packages/theme/src/cli/utilities/theme-environment/html.ts

```diff
@@ -67,6 +70,22 @@ export function getHtmlHandler(theme: Theme, ctx: DevServerContext) {
   })
 }
 
+function logRequestLine(event: H3Event, response: Response) {
+  const truncatedPath =
+    event.path.length > CHARACTER_TRUNCATION_LIMIT
+      ? `${event.path.substring(0, CHARACTER_TRUNCATION_LIMIT)}...`
+      : event.path
+  const serverTiming = response.headers.get('server-timing')
+  const requestDuration = serverTiming?.match(/cfRequestDuration;dur=([\d.]+)/)?.[1]
+  const durationString = requestDuration ? Math.round(Number(requestDuration)) : 'N/A'
+
+  outputInfo(
+    outputContent`• ${timestampDateFormat.format(new Date())} Request ${outputToken.raw('»')} ${outputToken.gray(
+      `${event.method} ${truncatedPath} ${durationString}ms`,
```

Comment:
I believe that we could have something like this here:
```ts
const durationString = requestDuration ? `${Math.round(Number(requestDuration))}ms` : ''
...
outputContent`• ${timestampDateFormat.format(new Date())} Request ${outputToken.raw('»')} ${outputToken.gray(
      `${event.method} ${truncatedPath} ${durationString}`,
```

This way we can prevent `N/Ams` from appearing :)

Reply:
Great catch. I'm making that change